### PR TITLE
[Papercut][SW-16034]Adding Feature Multiple pictures remove in Article Detail

### PR DESCRIPTION
**Pull request by TobiasWilming**
This Request adds, that you can now. Delete multiple Images at once in the articledetails. It adds multiple select in the list.js and a while loop which goes through the selection and removes the picture one by one

| Questions | Answers |
| --- | --- |
| BC breaks? | no |
| Tests pass? | yes |
| Related tickets? | https://issues.shopware.com/#/issues/SW-16034 |
| How to test? | Select Multiple images in article details with Ctrl and press the delete button |

### CODE REVIEW

**Comment by bcremer** [comment]
Hi @TobiasWilming, thanks for your Contribution. Can you please reformat the commit message.
Please see the following comment for a howto: https://github.com/shopware/shopware/pull/703#issuecomment-244731243.

Thanks.

**Comment by TobiasWilming** [comment]
done

File: themes/Backend/ExtJs/backend/article/controller/media.js [hunk 1]
```diff
@@ -377,16 +377,20 @@ Ext.define('Shopware.apps.Article.controller.Media', {
...
-            return false;
+        while(mediaList.getSelectionModel().selected.length){
+            if (mediaList.getSelectionModel() && mediaList.getSelectionModel().selected && mediaList.getSelectionModel().selected.first()) {
+                selected = mediaList.getSelectionModel().selected.items[0];
+            }
+
+            if (!(selected instanceof Ext.data.Model)) {
+                return false;
+            }
+            changeMain = (selected.get('main') === 1);
```
Comment (OliverSkroblin) [inline themes/Backend/ExtJs/backend/article/controller/media.js]: changeMain validiation won't works if last selected isn't the main image.

File: themes/Backend/ExtJs/backend/article/controller/media.js [hunk 2]
```diff
@@ -377,16 +377,20 @@ Ext.define('Shopware.apps.Article.controller.Media', {
             changeMain,
```
Comment (OliverSkroblin) [inline themes/Backend/ExtJs/backend/article/controller/media.js]: Could you simplify this sources.

```

onRemoveImage: function() {
        var me = this,
            mediaList = me.getMediaList(),
            store = mediaList.getStore();

        store.remove(mediaList.getSelectionModel().getSelection());

        var main = store.findExact('main', 1);
        var first = store.getAt(0);
        if (main < 0 && first instanceof Ext.data.Model)  {
            first.set('main', 1);
        }
    }

```

File: themes/Backend/ExtJs/backend/article/controller/media.js (hunk 1 above)
Comment (OliverSkroblin) [reply themes/Backend/ExtJs/backend/article/controller/media.js]: Please change the button snippet to "delete image(s)"

**Pull request merged**